scripts/vm/hypervisor/xenserver/xenserver84/vmops

```diff
@@ -0,0 +1,1607 @@
+#!/usr/bin/python
```

Comment:
suggestion - should this be /usr/bin/python3 (or use /usr/bin/env python3)? cc @Pearl1594 @harikrishna-patnala @weizhouapache

Reply:
`/usr/bin/env python3` looks good